### PR TITLE
Allow arbitrary arguments to commit command

### PR DESCRIPTION
**Pull request by lubomir**
Basically just take these and tack them on the end of commit command. One use case may be to generate a Signed-off-by line.

I couldn't get the test suite to run so far, so maybe this will break it. But it works when testing manually.

### CODE REVIEW

File: bumpversion/__init__.py [hunk 1]
```diff
@@ -75,14 +75,16 @@ def __call__(self, parser, namespace, values, option_string=None):
 class BaseVCS(object):
 
     @classmethod
-    def commit(cls, message):
+    def commit(cls, message, extra_args=[]):
```
[the remark this answers is not in the document]
Comment (lubomir) [reply bumpversion/__init__.py]: Fixed.

**Comment by c4urself** [comment]
Let's rebase and add to the documentation as well.

**Comment by florisla** [comment]
We probably also want to support this functionality from the config file.

**Comment by lubomir** [comment]
I rebased on latest master and added documentation to the README file. It already seems to work for me with a config file through argparse magic.

**Comment by florisla** [comment]
@ekhol @c4urself This looks ready to merge.

**Pull request merged**